### DISCUSSION TITLE
v2.0.6

### DISCUSSION THREAD

**Discussion by aaronhelton** [dag-hammarskjold-library/dlx-rest#1094] [General]
## What's Changed
* Add a preview record icon next to each search result by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1054
* Configure for Atlas by @aaronhelton in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1067
* Fix Atlas connection issue option in mongoengine connect by @aaronhelton in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1081
* Make speeches 710 not required by @aaronhelton in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1073
* Update README.md to remove sensitive information by @aaronhelton in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1085
* Fix an issue where not all copied fields were pasting by @aaronhelton in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1074
* Filter out votes and speeches from regular bib search by @aaronhelton in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1075
* Update auth config to include 530 and 591 by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1082
* Bump requests from 2.27.1 to 2.31.0 by @dependabot in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1086
* Add the ability to link files to records by ISBN by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1088

**Full Changelog**: https://github.com/dag-hammarskjold-library/dlx-rest/compare/v2.0.5...v2.0.6

<hr /><em>This discussion was created from the release <a href='https://github.com/dag-hammarskjold-library/dlx-rest/releases/tag/v2.0.6'>v2.0.6</a>.</em>

**Comment by aaronhelton** [comment]
Deployed to QAT 2023-05-25

cc @viola-v

**Comment by viola-v** [comment]
Tested in QAT 2023-05-25

Opened a new ticket based on #1075 because the result counts are off now, but that's not a priority.

Everything else looks good!
